src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -115,20 +113,17 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
     public isEnabled(type: BannerType): boolean {
         switch (type) {
             case BannerType.InsidersNotebookSurvey:
-                if (this.useVSCodeNotebookEditorApi && this.applicationEnvironment.channel === 'insiders') {
+                if (this.applicationEnvironment.channel === 'insiders' && this.useVSCodeNotebookEditorApi) {
                     return this.isEnabledInternal(type);
```

Comment:
Identical, just swapped right?

Reply:
yeah, I swapped it just so it looks the same as the other two